File: mau/2c275.pas
function scs(k : integer) : integer;
var s : integer;
begin
    s:=0;
    while k > 0 do begin
        inc(s);
        k:= k div 10;
    end;
    exit(s);
end;
var n, x, t : integer;
BEGIN
    readln(n);
    t:= 0;
    for x:= 1 to n do
        t += scs(x);
    write(t);
END.

File: mau/2c275.cpp
#include <iostream>
using namespace std;
int scs(int k) {
    int s = 0;
    while (k > 0) {
        s++;
        k/=10;
    }
    return s;
}
int main()
{
    int x, n, t = 0;
    cin >> n;
    for (x = 1; x <=n; x++)
        t += scs(x);
    cout<<t;
    return 0;
}
